_examples/ceph-source-example.adoc

```diff
@@ -17,35 +17,53 @@ This is an example of the key pieces needed to connect your
 application to data in Ceph using S3 API. It is presented as steps in a Jupyter
 notebook. To run this example, you need a Ceph environment running in your network
 and a running Spark cluster in your OpenShift project.
-Optionally, you can run in your own computer using `vagrant` and `ansible`
+Optionally, you can run in your own computer using `ceph-nano`
 (check the <<prerequisites>> section if you need to deploy a local instance of Ceph).
 
 https://github.com/radanalyticsio/radanalyticsio.github.io/blob/master/assets/ceph-source-example/ceph-example.ipynb[Ceph Source Example]
 
-https://github.com/ceph/ceph-ansible[Ceph Ansible playbooks]
+https://github.com/ceph/cn[Ceph Ansible playbooks]
 
 
 [[prerequisites]]
 == Prerequisites
 
-In case you need to configure your own Ceph environment, follow the
-steps below:
+In case you need to configure your own Ceph environment, follow the steps below:
 
-1. Clone https://github.com/ceph/ceph-ansible[ceph-ansible] GitHub repo.
-2. Rename `vagrant_variables.yml.sample` to `vagrant_variables.yml`
-   and edit the file by changing to the following snippet:
+1. Download `ceph-nano` in your computer:
 
-   ...
-   # DEFINE THE NUMBER OF VMS TO RUN
-   mon_vms: 1
-   osd_vms: 3
-   mds_vms: 0
-   rgw_vms: 1
-   ...
+  ...
```

Comment:
these `...` markup need to be removed as well. with the block indented they are already picking up the source block